ui/DesignSystem/Component/KeyHint.svelte

```diff
@@ -0,0 +1,33 @@
+<script>
+  import { isMac } from "./../../src/settings.ts";
+
+  export let hint = null;
+  export let noModifier = false;
+  export let style = null;
```

Comment:
```suggestion
  export let hint = "";
  export let noModifier: boolean = false;
  export let style = "";
```

Reply:
In case you're wondering why we have to have the `boolean` type annotation here and not for the `string`'s, it's an idiosyncrasy of svelte at this time and the compiler needs to know that it's a `boolean` instead of `false | undefined`. 

tldr; if it's an optional prop, we should specify a default value. If it's an optional prop that happens to be a `boolean`, we should also let the compiler know it's a `boolean`. If it's a required prop, e.g. something that the component should be passed every single time, we can just specify the type, à la:
```typescript
    export let alwaysProvideThisOrCompilerShouldComplain: string;
```

Reply:
Ok, that makes sense. I'm sure I'll have to go through the motions a couple of time to get it completely. Thanks for taking the time to explain it :)

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/914/commits/4da650c03cb5846cdd55d435f473cb6dde0749fd